api/types/constants.go

```diff
@@ -724,8 +724,8 @@ const (
 
 	// ReqAnnotationSchedulesLabel is the request annotation key at which schedules are stored for access plugins.
 	ReqAnnotationSchedulesLabel = "/schedules"
```

Comment:
Let's rename this one to `ReqAnnotationApproveSchedulesLabel` too then, for consistency.